### PR TITLE
Change `/all` to `/documents`

### PR DESCRIPTION
**Pull request by jeffdaley**
Moves the "All Docs" route from `/all` to `/documents` to give us a better framework to add new models, e.g., projects, products, teams, that can be browsed. References to the `/all` route are now redirected to `/documents`.

### CODE REVIEW

File: web/app/controllers/authenticated/all.ts [hunk 1]
```diff
@@ -1,8 +1,4 @@
 import Controller from "@ember/controller";
```
Comment (jfreda) [inline web/app/controllers/authenticated/all.ts:1]: Do we need this file anymore?

**Review by jfreda — APPROVED** [review]
Do you think we should add an acceptance test for the redirect? (not sure offhand if this is possible, but if so, it might be able to catch regressions)

File: web/app/controllers/authenticated/all.ts (hunk 1 above)
Comment (jeffdaley) [reply web/app/controllers/authenticated/all.ts:1]: It's needed to pass the queryParams to the `/documents` route on redirect. I've added a note to the controller.

**Comment by jeffdaley** [comment]
> Do you think we should add an acceptance test for the redirect?

Thanks for catching this. I wrote one in my spike branch but forgot to bring it over. ✅

**Pull request merged**